src/index.test.ts

```diff
@@ -60,16 +60,18 @@ test('export', (t): void => {
           '@typescript-eslint/no-non-null-assertion': 'error',
           '@typescript-eslint/no-object-literal-type-assertion': 'error',
           '@typescript-eslint/no-this-alias': ['error', { allowDestructuring: true }],
-          '@typescript-eslint/no-triple-slash-reference': 'error',
           '@typescript-eslint/no-unnecessary-type-assertion': 'error',
           '@typescript-eslint/no-unused-vars': ['error', { args: 'none' }],
           '@typescript-eslint/no-use-before-define': ['error', { functions: false, classes: false, variables: false, typedefs: false }],
           '@typescript-eslint/no-useless-constructor': 'error',
           '@typescript-eslint/no-var-requires': 'error',
           '@typescript-eslint/prefer-function-type': 'error',
+          '@typescript-eslint/prefer-readonly': 'error',
           '@typescript-eslint/promise-function-async': 'error',
           '@typescript-eslint/restrict-plus-operands': 'error',
           '@typescript-eslint/require-array-sort-compare': 'error',
+          '@typescript-eslint/strict-boolean-expressions': 'error',
+          '@typescript-eslint/triple-slash-reference': ['error', { lib: 'never', path: 'never', types: 'never' }],
```

Comment:
I have been using TypeScript for a while and never used a triple slash directive. It seems like a thing of the past and perhaps this rule can help keep it in the past.

Yet, precisely because I have never used it, I don't know whether there's a valid use case for it in modern TypeScript. Anyone?

Also, the question of breaking existing codes... The way I see it is that this shareable config is not yet popular and so this is our chance to make great choices, before it gets popular.

Reply:
@mightyiam Never used them before myself, but for example major parts of TypeScripts own typings are using that reference style. So maybe not so "out of the past"?

https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html

Reply:
@Dangoo, thank you for your input. Perhaps it was in a podcast that I've heard this—not sure. In TypeScript itself those are slowly replaced with imports. I remember learning that.